.travis.yml

```diff
@@ -0,0 +1,15 @@
+language:
+  node_js
+
+node_js:
+  - "7"
```

Comment:
package.json has node version 6.0.0 not sure if we want to keep our version at 7 or change what is in .travis.yml from 7 to 6.